internal/outputs/http.go

```diff
@@ -53,20 +44,16 @@ func postRequest(url string, key string, data []byte) {
 
 	resp, err := client.Do(req)
 	if err != nil {
-		load.Logrus.WithFields(logrus.Fields{
-			"err": err,
-		}).Error("http: failed to send")
+		load.Logrus.WithError(err).Error("http: failed to send")
 	}
 	defer resp.Body.Close()
 
-	if resp != nil {
-		if resp.StatusCode > 299 || resp.StatusCode < 200 {
-			load.Logrus.WithFields(logrus.Fields{
-				"code": resp.StatusCode,
-			}).Error("http: post failed")
-		}
-	} else {
-		load.Logrus.Error("http: response nil")
+	if resp == nil {
+		return errors.New("http: response nil")
```

Comment:
why do we use **errors.New** here instead of **fmt.Errorf**. Even if we don't anything to format, it's strange to use a different way to create the error